src/run/lib/Util_file.ml

```diff
@@ -30,6 +30,15 @@ let jsoo = ref false
    unclear.
 *)
 
+(* copied from `OSS/libs/commons/UFile.ml` *)
+(* Temporary files created using Python's [tempfile.NamedTemporaryFiles] on
+    Windows enables the [FILE_SHARE_DELETE] sharing mode. Files that have open
+    handles with the [FILE_SHARE_DELETE] sharing mode can only be re-opened in
+    that mode. To make sure we won't run into problems opening the file, we
+    add the [O_SHARE_DELETE] flag when opening all files. *)
+let win_safe_open_in_bin file : Unix.file_descr =
+  Unix.openfile file [ O_CREAT; O_RDONLY; O_SHARE_DELETE ] 0o666
```

Comment:
Is `Unix.openfile` jsoo-safe?  In `UFile.ml`, we have a `!jsoo then Stdlib.open_in_bin file else win_safe_open_in_bin file` check.  (I know we're getting rid of jsoo soon, so perhaps this doesn't matter.)

Reply:
Never mind!  I see `read_file` has a separate path for jsoo.

Reply:
This function is executed only in the non-`jsoo` path, see line 43 in the def of `read_file` below